reviewdog.go

```diff
@@ -21,11 +21,15 @@ type Reviewdog struct {
 	d        DiffService
 }
 
+var ExportedVarWithoutComment = 1 // TODO: revert
+
 // NewReviewdog returns a new Reviewdog.
 func NewReviewdog(toolname string, p Parser, c CommentService, d DiffService) *Reviewdog {
 	return &Reviewdog{p: p, c: c, d: d, toolname: toolname}
 }
 
+var unusedVar = 2 // TODO: revert
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var unusedVar is unused